packages/react/src/combobox/root/AriaCombobox.tsx

```diff
@@ -955,9 +960,7 @@ export function AriaCombobox<Value = any, Mode extends SelectionMode = 'none'>(
     // Floating UI tests don't require `role="row"` wrappers, so retains the number API.
     cols: grid ? 2 : 1,
     orientation: grid ? 'horizontal' : undefined,
-    disabledIndices: virtualized
```

Comment:
Why was this logic changed?

Reply:
It's not actually necessary, but I was debugging an issue with the virtualized demos which I PR'd here: https://github.com/mui/base-ui/pull/3066